### PR TITLE
[v15] docs: fix version number

### PR DESCRIPTION
**Pull request by mmcallister**
Due to the backport https://github.com/gravitational/teleport/pull/43397, the version number displayed is incorrect

### CODE REVIEW

**Review by stevenGravy — APPROVED** [review]

File: docs/config.json [hunk 1]
```diff
@@ -1489,6 +1771,10 @@
           "title": "Active Directory",
           "slug": "/desktop-access/active-directory/"
         },
+        {
+          "title": "Active Directory (Manual)",
```
Comment (zmb3) [inline docs/config.json:1775]: Did this get removed by mistake?

File: docs/config.json [hunk 2]
```diff
@@ -2021,6 +2318,316 @@
       "destination": "/kubernetes-access/introduction/",
       "permanent": true
     },
+    {
```
Comment (zmb3) [inline docs/config.json:2321]: What are all these extra redirects?

**Comment by zmb3** [comment]
I'm having a hard time understanding if this is correct or not because there are a lot more changes than just fixing a version number. Can you help clarify?

**Comment by mmcallister** [comment]
@zmb3 So it looks like I had the wrong copy of the config file (If I'm reading this correctly it actually looks like I was putting the newer config file from Master into the v16 branch change) - here are my original changes yesterday:

https://github.com/gravitational/teleport/commit/5a5e04bcf8f12e700a7d3bdaf994a42ed7a69032

So now aside from my entries for the new content, the rest of this should revert back to the way it was beforehand.

**Comment by mmcallister** [comment]
@zmb3 \ it looks like those are intended from looking at git blame. Lines 663 - 651 are my changes.

**Review by zmb3 — APPROVED** [review]

**Pull request merged**